ttnn/cpp/ttnn/operations/data_movement/reshape_view/reshape.cpp

```diff
@@ -111,7 +111,7 @@ ttnn::Tensor ReshapeViewOperation::invoke(const ttnn::Tensor& tensor, const ttnn
     }
 
     bool tile_tensor_view_reshape_possible = (layout == ttnn::Layout::TILE and
-        ((shape.with_tile_padding()[-2] % ttnn::TILE_SIZE == 0) and (shape.with_tile_padding()[-1] % ttnn::TILE_SIZE == 0)) and
+        ((shape.with_tile_padding().rank() < 2 or shape.with_tile_padding()[-2] % ttnn::TILE_SIZE == 0) and (shape.with_tile_padding()[-1] % ttnn::TILE_SIZE == 0)) and
         (tensor_shape.with_tile_padding()[-1] == shape.with_tile_padding()[-1])
         );
 
```

Comment:
@jvegaTT could you look at this?

Reply:
I don't think this is valid. Say I reshape {2,1[32],16[32]} to [32] which is a valid reshape. This goes from needing 2 tiles to needing 1 so a view is not legal. But 32 in tile is really {1,1[32],32[32]} and the above logic would have said that a view is possible. Then the tensor.reshape will assert a failure.

Reply:
On update - that's better but I will need to do a better fix for the second case later. I will open up a ne issue for that to remember

Reply:
Tiny readability thing, you now have:
cond a and
(cond b and cond c) and
cond d

I think it will be easier to follow as
cond a and
cond b and 
cond c and
cond d

I know the old code did this but just a thought though not necessary